en/ui-system/components/editor/webview.md

```diff
@@ -0,0 +1,210 @@
+# WebView Component Reference
+
+WebView is a component for displaying web pages, you could use this component to embed a mini web browser in your games. Because different platforms have different authorization, API and control methods for WebView component. And have not yet formed a unified standard, only **Web**, **iOS**, and **Android** platforms are currently supported.
+
+![webview](./webview/webview.png)
+
+Click **Add Component** at the bottom of **Properties** panel and select **WebView** from **UI Component** to add the WebView component to the node.
+
+For more information about WebView API, please refer to [WebView API](../../../api/en/classes/WebView.html) for details.
```

Comment:
For more information about WebView API, please refer to [WebView API](../../../api/en/classes/WebView.html) for details.
-->
For more information about WebView API, please refer to [WebView API](../../../api/en/classes/WebView.html) documentation.